### PR TITLE
added some files

### PR DESCRIPTION
**Pull request by Catalin12**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+</html>
```
Comment (danielursu) [inline index.html]: do this work? :))

Comment (danielursu) [reply index.html]: move the head and body into html tag

File: word-counter.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+class WordCounter extends HTMLParagraphElement {
+	constructor() {
+		super();
+		this.attachShadow({ mode: 'open' });
+		this.shadowRoot.appendChild(templateCountWords.content.cloneNode(true));
+	}
+
+	revealCount() {
+		let numberOfWords = this.shadowRoot.querySelector('p').innerText.split(" ").length;
```
Comment (danielursu) [inline word-counter.js]:
```suggestion
		const numberOfWords = this.shadowRoot.querySelector('p').innerText.split(" ").length;
```

if you do not modify the variable it is better to declare it as const, not as let

**Review by danielursu — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Catalin12) [reply index.html]: lol :))

File: index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+				avatar="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQm6L2jdgUcljnfqT2QraTGxE3rXXZGDk-SFQ&usqp=CAU">
+			</user-avatar>
+		</div>
+		<div>
+			<h2>Task #2.</h2>
+			<live-timer id="live-timer-id"></live-timer>
+		</div>
+		<div>
+			<h2>Task #3.</h2>
+			<word-counter></word-counter>
```
Comment (danielursu) [inline index.html]: send to word-counter the text via attribute "text"
```suggestion
			<p is="word-counter" text="Acesta este un text simplu. Cate cuvinte am?"></p>
```

**Review by danielursu — CHANGES_REQUESTED** [review]

File: word-counter.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+const templateCountWords = document.createElement('template');
+templateCountWords.innerHTML = `
+	<style>
+		.word-counter p {
+			border-style: solid;
+			border-width: 1px;
+		}
+	</style>
+	<div class="word-counter">
+		<p class="word-counter">Acesta este un simplu text.</p>
```
Comment (danielursu) [inline word-counter.js]: get rid of this line. You do not need anymore

**Review by danielursu — CHANGES_REQUESTED** [review]

**Review by danielursu — APPROVED** [review]

**Pull request merged**